src/dodal/devices/electron_analyser/base_analyser.py

```diff
@@ -0,0 +1,107 @@
+import asyncio
+from typing import Generic, TypeVar
+
+from bluesky.protocols import Movable
+from ophyd_async.core import AsyncStatus, StandardReadable
+from ophyd_async.epics.core import epics_signal_rw
+
+from dodal.devices.electron_analyser.base_region import EnergyMode, TBaseRegion
+from dodal.log import LOGGER
+
+
+class BaseAnalyser(StandardReadable, Movable, Generic[TBaseRegion]):
+    """
+    Generic device to configure electron analyser with new region settings.
+    Electron analysers should inherit from this class if they need further
+    specialisation.
+    """
+
+    PV_LENS_MODE = "LENS_MODE"
+    PV_PASS_ENERGY = "PASS_ENERGY"
+    PV_ACQISITION_MODE = "ACQ_MODE"
+    PV_LOW_ENERGY = "LOW_ENERGY"
+    PV_HIGH_ENERGY = "HIGH_ENERGY"
+    PV_SLICES = "SLICES"
+    PV_ITERATIONS = "NumExposures"
+    PV_ENERGY_STEP = "STEP_SIZE"
+
+    def __init__(self, prefix: str, name: str = "") -> None:
+        self.prefix = prefix
+
+        with self.add_children_as_readables():
+            self.low_energy_signal = epics_signal_rw(
+                float, self.prefix + BaseAnalyser.PV_LOW_ENERGY
+            )
+            self.high_energy_signal = epics_signal_rw(
+                float, self.prefix + BaseAnalyser.PV_HIGH_ENERGY
+            )
+            self.slices_signal = epics_signal_rw(
+                int, self.prefix + BaseAnalyser.PV_SLICES
+            )
+            self.lens_mode_signal = epics_signal_rw(
+                str, self.prefix + BaseAnalyser.PV_LENS_MODE
+            )
+            self.pass_energy_signal = epics_signal_rw(
+                self.get_pass_energy_type(), self.prefix + BaseAnalyser.PV_PASS_ENERGY
+            )
+            self.energy_step_signal = epics_signal_rw(
+                float, self.prefix + BaseAnalyser.PV_ENERGY_STEP
+            )
+            self.iterations_signal = epics_signal_rw(
+                int, self.prefix + BaseAnalyser.PV_ITERATIONS
+            )
+            self.acquisition_mode_signal = epics_signal_rw(
+                str, self.prefix + BaseAnalyser.PV_ACQISITION_MODE
+            )
```

Comment:
Should: I think the word `_signal` is redundant in these definitions. `abs_set(analyser.high_energy, 1000)` is just as readable as `abs_set(analyser.high_energy_signal, 1000)` and shorter. Other devices in dodal do not specify `signal` everywhere